docs/HowTo/Run-Docker-Image.md

```diff
@@ -0,0 +1,88 @@
+description: Run Orion using the official docker image
+<!--- END of page meta data -->
+
+# Running the Orion Docker Image
+
+Use the Orion Docker image to run a single Orion node without installing Orion.
+
+__Prerequisites__:
+
+* [Docker](https://docs.docker.com/install/)
+
+!!! important
+    Do not mount a volume at the (`/opt/orion`) path, this prevents
+    Orion from safely launching.
+
+    In this example the required volume is mounted at the `/data` path.
+
+In this example we create the [password file](#2-create-password-file), [public/private keypair](#3-generate-keys),
+and [Orion configuration file](#4-create-a-configuration-file) in the
+`/Users/user1/Node1/Orion` directory.
+
+!!! note
+    On macOS the default home directory is `/Users/<username>`, and on Linux it
+    is usually `/home/<username>`.
+
+## 1. Create the Directory Structure
+
+Create the sub-directories in the home directory.
+
+``` bash
+mkdir -p Node1/Orion
+```
+
+## 2. Create Password File
+
+Create a file containing the password used to encrypt the key pair.
+
+Specify the password when [generating the keys](#3-generate-keys), the
+file is specified for the `passwords` property in the [configuration file](#4-create-a-configuration-file).
+
+## 3. Generate Keys
+
+Generate a public/private keypair for the Orion node using Docker:
+
+``` bash
+docker run -i --rm --mount type=bind,source=/Users/user1/Node1/Orion,target=/data pegasyseng/orion:latest -g /data/testKey
+```
+
+At the prompt, enter the [password](#2-create-password-file) to encrypt the
+key pair.
+
+The public/private key pair is generated and the keys saved in the `nodeKey.pub`
+and `nodeKey.key` files.
+
+## 4. Create a Configuration File
+
+Create a file called `orion.conf` and add the following [properties](../Reference/Configuration-File.md):
+
+```
```

Comment:
Can we specify the config file syntax scheme? It looks like TOML so it may be nice to add this to the code block and explicitely write it so users ca configure their ide to check the syntax. It may be a change to make in /Reference/Configuration-File.md

Reply:
Added that detail to /Reference/Configuration-File.md